kano_updater/ui/media/flappy-judoka/src/core/gamestate.py

```diff
@@ -383,6 +385,14 @@ def __init__(self, pipes, score):
 
         self.pipes.reset_pipes()
 
+        try:
+            # TODO: track best score as well
+            track_data('updater-flappy-judoka', {
```

Comment:
I'd call the event 'updater-flappy-judoka-score'. Because there can be more data events per app.